DESCRIPTION

```diff
@@ -1,7 +1,8 @@
 Package: ANTsR
 Type: Package
 Title: ANTs in R
-Version: 0.3.1
+Version: 0.3.2
```

Comment:
Just a thought ... some people adopt a versioning scheme for dev branches that might have this be `0.3.1.99`. Then you might only release on master and it's easy to see that you have a dev version that is on its way to becoming `0.3.2`. There are plenty of alternative approaches of course including:

* http://yihui.name/en/2013/06/r-package-versioning/
* http://semver.org/